packages/denylist-bot/src/services/consumer.ts

```diff
@@ -0,0 +1,91 @@
+import type { Env } from "../types/env";
+import type { QueuedData } from "../types/pagination";
+import { setupNear, viewFunction, AttachedGas } from "../utils";
+
+const riskMap: any = {
+  "Scam": 5,
+  "Theft": 5,
+  "Sanctions": 5,
+  "ChildAbuse": 5,
+  "Ransomware": 5,
+  "Counterfeit": 5,
+  "DarknetService": 5,
+  "TerroristFinancing": 5,
+  "IllicitOrganization": 5,
+};
+
+/** fetchRiskParams
+ * This function must perform a view call to contract to fetch
+ * there is currently no such view call in the contract. Mock
+ * @param category The risk category
+ * @returns Returns the max risk accepted by the contract
+ */
+const fetchRiskParams = async (category: string): Promise<number> => {
```

Comment:
Risk params can now be fetched from contract using `view_risk_params` view call. Substitute current hard coded values for this view call.

View call can be checked on main. Rebase to have it in this branch